.github/workflows/rust.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - market-decentralized/errors-experiments
```

Comment:
what is it needed for?

Reply:
Don't worry. This branch isn't meant to be ever merged to master ;)